.github/workflows/tests.yml

```diff
@@ -0,0 +1,179 @@
+name: tests
```

Comment:
Would love for this to follow common naming patterns; perhaps `acceptance-test`
like here:
https://github.com/pulumi/pulumi-awsx/blob/master/.github/workflows/run-acceptance-tests.yml#L140